### PR TITLE
Fix a minor documentation grammar

### PR DESCRIPTION
**Pull request by anmolbabu**
Fix a minor documentation grammar as per
https://github.com/redhat-developer/odo/pull/741/files#r227419998

Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

File: cmd/config.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ var configurationCmd = &cobra.Command{
 
 Available Parameters:
 UpdateNotification - Controls if an update notification is shown or not (true or false)
-NamePrefix - Uses the value of this configurable as a prefix for naming app/component.If set to empty string, the name will be auto generated based on name of component/component source/current directory`,
+NamePrefix - Default prefix is the current director name. Use this value to set a default name prefix`,
```
Comment (kadel) [inline cmd/config.go]: "directory name"

Comment (anmolbabu) [reply cmd/config.go]: Ack Done

**Comment by mik-dass** [comment]
@kadel Can we merge this?

**Comment by anmolbabu** [comment]
Merging based on 3 acks and oral ack from @kadel :)

**Pull request merged**